mewarpx/examples/capacitive_discharge.py

```diff
@@ -20,173 +20,182 @@
+        #######################################################################
+
+        # self.solver = picmi.ElectrostaticSolver(
+        #    grid=mwxrun.grid, method='Multigrid', required_precision=1e-12
+        # )
+        self.solver = poisson_pseudo_1d.PoissonSolverPseudo1D(grid=mwxrun.grid)
+        mwxrun.simulation.solver = self.solver
+
+        #######################################################################
+        # Particle types setup                                                #
+        #######################################################################
+
+        self.electrons = mespecies.Species(
+            particle_type='electron', name='electrons'
+        )
+        self.ions = mespecies.Species(
+            particle_type='He', name='he_ions', charge='q_e'
+        )
 
-#####################################
-# grid, solver and timesteps
-#####################################
+        #######################################################################
+        # Collision  initialization                                           #
+        #######################################################################
 
-mwxrun.init_grid(xmin, xmax, zmin, zmax, nx, nz)
-mwxrun.grid.potential_zmax = anode_voltage
+        self.mcc = mcc_wrapper.MCC(
+            self.electrons, self.ions, T_INERT=self.T_INERT,
+            N_INERT=self.N_INERT, exclude_collisions=['charge_exchange']
+        )
 
-# solver = picmi.ElectrostaticSolver(
-#    grid=mwxrun.grid, method='Multigrid', required_precision=1e-12
-# )
-solver = poisson_pseudo_1d.PoissonSolverPseudo1D(grid=mwxrun.grid)
+        #######################################################################
+        # Neutral plasma injection                                            #
+        #######################################################################
 
-mwxrun.simulation.solver = solver
-mwxrun.init_timestep(DT=DT)
-mwxrun.simulation.max_steps = max_steps
+        self.vol_emitter = emission.UniformDistributionVolumeEmitter(
+            T=self.T_ELEC
+        )
 
-##########################
-# declare species
-##########################
+        self.plasma_injector = emission.PlasmaInjector(
+            emitter=self.vol_emitter, species1=self.electrons,
+            species2=self.ions, npart=2 * self.SEED_NPPC * self.NX * self.NZ,
+            T_2=self.T_INERT, plasma_density=self.PLASMA_DENSITY
+        )
 
-electrons = mespecies.Species(particle_type='electron', name='electrons')
-ions = mespecies.Species(particle_type='He', name='he_ions', charge='q_e')
+        #######################################################################
+        # Initialize run and print diagnostic info                            #
+        #######################################################################
 
-##########################
-# neutral plasma injection
-##########################
+        mwxrun.init_run()
```

Comment:
Is this going to be a strictly Modern Electron version of the script, diverging from upstream? If so, it would be good to give it a different name. If not, we can't well use `mwxrun` and the like.

Reply:
Oops, this is the one under mewarpx